completions/strace

```diff
@@ -40,9 +40,9 @@ _strace()
                             while read -r define syscall rest; do
                                 [[ $define == "#define" &&
                                     $syscall =~ ^__NR_(.+) ]] &&
-                                    syscalls[${BASH_REMATCH[1]}]=1
+                                    syscalls[${BASH_REMATCH[1]}]=true
                             done 2>/dev/null </usr/include/asm/unistd.h
-                            if [[ ! $syscalls ]]; then
+                            if ((${#syscalls[@]} == 0)); then
```

Comment:
Cherry-picked as 223aa08665959fcecc4c0191f7a5bcc44d3cf05c

Reply:
Thank you!